src/Credential/KYCTypes.ts

```diff
@@ -7,12 +7,20 @@ const sortKYCType = (type: string): string => type.split("+").sort().join("+");
 const SupportedKYCs = [
   "plus+liveness+wallet",
   "plus+liveness+wallet+sow",
+  "plus+liveness+wallet-eth",
```

Comment:
you can only append to this list because the credential kyc type is the index of the level on this list